canonical-web-and-design/jp.ubuntu.com
# Update pepabo engage page

## Done

- Update first paragraph to match copy doc updates
- Remove 'pepabo' text from logo engage page and home page
- Fix quotation marks as not to obscure text

## QA

- Check out this feature branch
- Run the site using the command ./run serve or dotrun
- View the site locally in your web browser at: http://0.0.0.0:8012/engage/case-study-gmo-- pepabo and check updated paragraph from [copy doc](https://docs.google.com/document/d/1TewivXyiRIQyz8m4IXg1o1gC7nsZAxrfcrB32szE8jI/edit#heading=h.gjdgxs)
- Check quotation marks on the block quote

## Issue / Card

Resolving these [requests](https://github.com/canonical-web-and-design/jp.ubuntu.com/pull/238#issuecomment-704111409)

## Screenshots

![image](https://user-images.githubusercontent.com/58959073/95193527-d9470d80-07cb-11eb-9f2b-46b8c9fdf164.png)
